### ISSUE TITLE
Using the generator outside of the repo

### ISSUE THREAD

**Issue by stefanorosss** [lucidrains/lightweight-gan#39]
Hi, I'm trying to apply closed-form factorization of the weights to explore the latent space of a model I have trained.
Unfortunately, using anything outside of the module itself doesn't work as expected.

I've installed the repo and so I am able to run every command using 'lightweight_gan' as a command with no problems, I can generate interpolations, grid images etc.
Whenever I try to do the same outside of the repo (importing the Generator or Trainer class, or some functions) I can't reproduce the results.
Two things to notice: 
1. The image generated is not completely random, I can see that the badly-generated image (right) share some resemblance with the well-generated images (left)
![45](https://user-images.githubusercontent.com/60915557/103944769-75b6de00-5134-11eb-8998-e853b8191ef1.jpg)![generated-01-07-2021_21-08-56](https://user-images.githubusercontent.com/60915557/103944792-7c455580-5134-11eb-87a0-b94e4a95beed.jpg)
2. My generator has missing layers (maybe?)
`train = Trainer(name=args.modeltype, models_dir=args.weights, attn_res_layers=[32,64], image_size=256)
train.load(325)
print(train.GAN.G)`
gives me this 
<details> 
  <summary> Generator</summary>
   Generator(

  (initial_conv): Sequential(
    (0): ConvTranspose2d(256, 512, kernel_size=(4, 4), stride=(1, 1))
    (1): BatchNorm2d(512, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
    (2): GLU(dim=1)
  )
  (layers): ModuleList(
    (0): ModuleList(
      (0): Sequential(
        (0): Upsample(scale_factor=2.0, mode=nearest)
        (1): Identity()
        (2): Conv2d(256, 1024, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
        (3): BatchNorm2d(1024, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
        (4): GLU(dim=1)
      )
      (1): None
      (2): None
      (3): None
    )
    (1): ModuleList(
      (0): Sequential(
        (0): Upsample(scale_factor=2.0, mode=nearest)
        (1): Identity()
        (2): Conv2d(512, 1024, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
        (3): BatchNorm2d(1024, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
        (4): GLU(dim=1)
      )
      (1): SLE(
        (avg_pool): AdaptiveAvgPool2d(output_size=(4, 4))
        (max_pool): AdaptiveMaxPool2d(output_size=(4, 4))
        (net): Sequential(
          (0): Conv2d(1024, 256, kernel_size=(4, 4), stride=(1, 1))
          (1): LeakyReLU(negative_slope=0.1)
          (2): Conv2d(256, 64, kernel_size=(1, 1), stride=(1, 1))
          (3): Sigmoid()
        )
      )
      (2): None
      (3): None
    )
    (2): ModuleList(
      (0): Sequential(
        (0): Upsample(scale_factor=2.0, mode=nearest)
        (1): Identity()
        (2): Conv2d(512, 512, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
        (3): BatchNorm2d(512, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
        (4): GLU(dim=1)
      )
      (1): SLE(
        (avg_pool): AdaptiveAvgPool2d(output_size=(4, 4))
        (max_pool): AdaptiveMaxPool2d(output_size=(4, 4))
        (net): Sequential(
          (0): Conv2d(512, 128, kernel_size=(4, 4), stride=(1, 1))
          (1): LeakyReLU(negative_slope=0.1)
          (2): Conv2d(128, 32, kernel_size=(1, 1), stride=(1, 1))
          (3): Sigmoid()
        )
      )
      (2): None
      (3): None
    )
    (3): ModuleList(
      (0): Sequential(
        (0): Upsample(scale_factor=2.0, mode=nearest)
        (1): Identity()
        (2): Conv2d(256, 256, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
        (3): BatchNorm2d(256, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
        (4): GLU(dim=1)
      )
      (1): None
      (2): None
      (3): Rezero(
        (fn): GSA(
          (to_qkv): Conv2d(256, 1536, kernel_size=(1, 1), stride=(1, 1), bias=False)
          (to_out): Conv2d(512, 256, kernel_size=(1, 1), stride=(1, 1))
        )
      )
    )
    (4): ModuleList(
      (0): Sequential(
        (0): Upsample(scale_factor=2.0, mode=nearest)
        (1): Identity()
        (2): Conv2d(128, 128, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
        (3): BatchNorm2d(128, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
        (4): GLU(dim=1)
      )
      (1): None
      (2): None
      (3): None
    )
    (5): ModuleList(
      (0): Sequential(
        (0): Upsample(scale_factor=2.0, mode=nearest)
        (1): Identity()
        (2): Conv2d(64, 64, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
        (3): BatchNorm2d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
        (4): GLU(dim=1)
      )
      (1): None
      (2): None
      (3): None
    )
  )
  (out_conv): Conv2d(32, 3, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
)
</details>

Any idea of what is happening? It feels like it is a simple problem but I can't make it work...
